docs/private-networks/how-to/monitor/chainlens.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Use Chainlens Explorer
+sidebar_position: 7
+description: Use Chainlens Explorer on a privacy-enabled Besu network
+tags:
+  - private networks
+---
+
+# Use Chainlens Blockchain Explorer
+
+[Chainlens Blockchain Explorer](https://chainlens.com/) supports public and private EVM networks.
+This page describes how to use the free version of Chainlens with its built-in support for
+[privacy-enabled](../../concepts/privacy/index.md) Besu networks created using the
+[Developer Quickstart](../../tutorials/quickstart.md).
+
+Chainlens provides an overview of the entire network, including block information, contract
+metadata, transaction searches, and [more](https://chainlens.com/).
+
+:::note
+You must connect to one of the privacy nodes (for example, `member1besu`), not the dedicated RPC, to
+allow access for Besu [privacy API methods](../../reference/api/index.md#priv-methods).
+In production networks, you must [secure access](../../../public-networks/how-to/use-besu-api/authenticate.md)
+to RPC nodes.
+:::
+
+## Prerequisites
+
+[Docker and Docker Compose](https://docs.docker.com/compose/install/) installed.
+
+## Start Chainlens
+
+Clone the [Chainlens GitHub repository](https://github.com/web3labs/chainlens-free):
+
+```bash
+git clone https://github.com/web3labs/chainlens-free
+```
+
+The repository contains a `docker-compose` directory to allow Chainlens to start with a Developer
+Quickstart test network.
+From the `docker-compose` directory, run the following command:
+
+```bash
+NODE_ENDPOINT=http://rpcnode:8545 docker-compose -f docker-compose.yml -f sirato-extensions/docker-compose-quorum-dev-quickstart.yml up
+```
+
+This command does two things:
+
+- Sets up the node endpoint
+- Tells Docker to run by using the two Docker Compose files provided
+
+The first `docker-compose.yml` file in the command contains all the services required for Chainlens.
+
+The second file named `docker-compose-besu` contains the network settings required to start
```

Comment:
Made some content edits to this page. Looks good to me, just need the file in this line to match up with the file in the corresponding example.

Reply:
fixed